src/github.com/getlantern/yamlconf/yamlconf_test.go

```diff
@@ -184,17 +185,26 @@ func TestCustomPoll(t *testing.T) {
 		Version: 2,
 		N: &Nested{
 			S: "Modified",
-			I: FIXED_I,
+			I: I,
 		},
 	}, updated, "Custom polled config should contain correct data")
 }
 
-func assertSavedConfigEquals(t *testing.T, file *os.File, expected *TestCfg) {
+func assertSavedConfigEquals(t *testing.T, m *Manager, file *os.File, expected *TestCfg) {
 	b, err := yaml.Marshal(expected)
 	if err != nil {
 		t.Fatalf("Unable to marshal expected to yaml: %s", err)
 	}
-	bod, err := ioutil.ReadFile(file.Name())
+	stream, err := m.obfuscationStream()
```

Comment:
With this, we are now only testing the obfuscated version path. Perhaps adding both would improve coverage. What do you think?